src/Heading.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const Heading = () => {
+  return (
+    <header className="App-header">
+      CYF Hotel
+      <div className="App-logo">
```

Comment:
I noticed that the hotel logo is spinning! How could you fix this?

Reply:
I removed animation.